test/tests/fvkernels/fv_coupled_var/coupled.i

```diff
@@ -49,6 +50,7 @@
   [rxn]
     type = FVReaction
     variable = v
+    rate = 2.0
```

Comment:
Did you manually verify the results that you are regolding here to ensure they are correct?

Reply:
I did. This test allows you to double check that the FV and FE results are consistent (and they are). Separately, I've found that the results match the expected pen-and-paper value for a source-decay problem.